src/com/farzaneh/model/Person.java

```diff
@@ -0,0 +1,74 @@
+package com.farzaneh.model;
+
+public class Person {
+	private Integer personId;
+	private String firstName;
+	private String lastName;
+	private String emailAddress;
+	private String principalName;
+	private String personTypeCode;
+
+	public Person() {
+		super();
+	}
+
+	public Person(Integer personId, String firstName, String lastName, String emailAddress, String principalName,
+			String personTypeCode) {
+		super();
```

Comment:
you can remove super() here , while Person has no direct parent

Reply:
Do it for your all other entities

Reply:
done